test/Datadog.Trace.ClrProfiler.IntegrationTests/LoadTests/LoadTestBase.cs

```diff
@@ -147,7 +147,7 @@ protected void RunLoadTestPart(LoadTestPart loadTestPart)
                 startInfo = new ProcessStartInfo(executable, $"{commandLineArgs}");
             }
 
-            environmentHelper.SetEnvironmentVariables(agentPort, aspNetPort, executable, startInfo.EnvironmentVariables);
+            environmentHelper.SetEnvironmentVariables(agentPort, aspNetPort, executable, false, startInfo.EnvironmentVariables);
```

Comment:
nit, for readability
```suggestion
            environmentHelper.SetEnvironmentVariables(agentPort, aspNetPort, executable, netStandardEnabled: false, startInfo.EnvironmentVariables);
```